frontend/src/scenes/max/maxGlobalLogic.ts

```diff
@@ -1,14 +1,13 @@
 import { actions, connect, kea, listeners, path, reducers, selectors } from 'kea'
+
 import { OrganizationMembershipLevel } from 'lib/constants'
 import { organizationLogic } from 'scenes/organizationLogic'
 
-import { AssistantContextualTool } from '~/queries/schema/schema-assistant-messages'
-
 import type { maxGlobalLogicType } from './maxGlobalLogicType'
 
 export interface ToolDefinition {
     /** A unique identifier for the tool */
-    name: AssistantContextualTool
```

Comment:
why are we removing the typing?

Reply:
Oops, I did this in the first hacky draft, don't remember why. Reverted